### PR TITLE
improvement: adds reply-to

### LINKED ISSUE #45: Add a Reply-To header
Hi,

It seems that we are struggling to add a "Reply-To" header (which is different from the "From" header). What is the best way to define a "Reply-To"? 

```
const message = createMimeMessage();
message.setHeader('Reply-To', contact.email);
```

generate the following message:

```
MIMETEXT_INVALID_HEADER_VALUE: You specified an invalid value for the header Reply-To
```

Thanks!

### PR DESCRIPTION
**Pull request by LarsFronius**
This is to fix #45 .
@muratgozel it looks like you may have some integration tests set up? Perhaps worth checking with them that this actually works.
Let me know if anything else is needed.

### CODE REVIEW

**Comment by jccr** [comment]
@muratgozel I think this will fix this issue I'm encountering https://github.com/muratgozel/MIMEText/issues/45

Would you consider merging this in? I can offer help if needed to push this along

File: tests/MIMEMessageHeader.spec.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ test('sets and reads headers', () => {
     a.set('To', new Mailbox('to@test.com'))
     a.set('Cc', [new Mailbox('cc@test.com'), new Mailbox('cc2@test.com')])
     a.set('Bcc', [new Mailbox('bcc@test.com'), new Mailbox('bcc2@test.com')])
+    a.set('Reply-To', [new Mailbox('reply-to@test.com')])
```
Comment (jccr) [inline tests/MIMEMessageHeader.spec.js:46]: Would it be worth to add a test for multiple mailboxes in the array?

**Review by jccr — APPROVED** [review]
Looks like a clean and clear contribution that fixes the `Reply-To` header validation and serialization.

Here's my code review approval, for what it's worth.

**Comment by muratgozel** [comment]
hi @LarsFronius sorry for the delay in the related issue. i think it will be more consistent if i just export `Mailbox` class and make everyone use of it when they try to set any header email address related.

this is what i mean: `message.setHeader("Sender", new Mailbox('...'))`

therefore closing this pr and will be publishing the update shortly.

**Pull request closed without merging**